.github/workflows/ci.yml

```diff
@@ -3,66 +3,66 @@ jobs:
     name: Are the types wrong?
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - uses: actions/checkout@v4
```

Comment:
Migration test is failing on this. Must have been a auto-merge of a failing PR. _Sigh_ this again...